### PR TITLE
Get user profile

### PR DESCRIPTION
**Pull request by RafailFerre**
Create userGet.spec and userHelper.
Added PmReports into .gitignore

### CODE REVIEW

**Review by ZdanovichOleg — DISMISSED** [review]

File: test/user/userGet.spec.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      await userHelper.register(userHelper.userData);
+      resLogin = await userHelper.login(userHelper.userData.email, userHelper.userData.password);
+      userId = resLogin.body.payload.userId;
+      resGet = await userHelper.getUser(userId);
+      // console.log(res);
+      // console.log(resGet.body);
+    });
+
+    after(async () => {
+      await userHelper.deleteUser(userId);
```
Comment (dumpdusty) [inline test/user/userGet.spec.js:18]: does `deleteUser` works ?!

File: test/user/userGet.spec.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    after(async () => {
+        await userHelper.deleteUser(userId);
+    });
+
+    it('verify status code', async () => {
+      expect(resGet.status).to.eq(400);
+    });
+
+    it('verify response message', async () => {
+      expect(resGet.body.message).to.eq('Permission denied');
```
Comment (dumpdusty) [inline test/user/userGet.spec.js]: looks like incorrect ER, please recheck

File: test/user/userGet.spec.js (hunk 1 above)
Comment (dumpdusty) [inline test/user/userGet.spec.js]: this one looks like a bug, as far as I understand user would not be able to delete an account

File: test/user/userGet.spec.js (hunk 2 above)
Comment (RafailFerre) [reply test/user/userGet.spec.js]: fixed

File: test/user/userGet.spec.js (hunk 1 above)
Comment (RafailFerre) [reply test/user/userGet.spec.js]: updated

**Review by dumpdusty — APPROVED** [review]
Great job !

**Review by ZdanovichOleg — APPROVED** [review]

**Pull request merged**